src/test/resources/test.properties

```diff
@@ -0,0 +1,1359 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
This is a rather lengthy properties file. Is it possible to condense/simplify this?

Reply:
This is the full properties file from jmeter. We had thought it would be useful to have annotations and default properties easily available. Alot of it can be removed with out having any effect on functionality, is it worth it to remove this information in favor of simplicity?

Reply:
I'm not entirely sure actually. I suppose we can leave it for now.